### ISSUE TITLE
Add algorithms of Cipher

### ISSUE THREAD

**Issue by Dark-System** [Dark-System/Cipher#1]
Anyone can take up this issue and contribute.
No barrier of Programming Language. Use whatever language you are comfortable in.

The Ciphers that should be included are :
1. Caeser Cipher
2. Playfair Cipher
3. Railfence Cipher
4. Hill Cipher
5. Monoalphabetic Cipher

Make sure to follow Coding conventions.

**Comment by saloniankita** [comment]
Can I take up the Caeser Cipher one?
Kindly assign me.

**Comment by Dark-System** [comment]
Assigned. You can start contributing.

**Comment by jangragaurav** [comment]
Assign me this please.

**Comment by Dark-System** [comment]
@saloniankita @sam0hack @jangragaurav Can you guys work on the documentation of the respective files that you added?
Make sure to add the description and explain the implementation if you do.

**Comment by saloniankita** [comment]
Yeah sure @Dark-System 
Do we have to include the references as well?

**Comment by Dark-System** [comment]
It's up to you. Adding the implementation part is mandatory.

**Comment by sam0hack** [comment]
Sure @Dark-System

**Comment by hryxna** [comment]
Hey, @Dark-System I have implemented Monoalphabetic cipher for the CS50 course.
Well it works quite well the only thing is I used cs50 libraries they mentioned us to use so if you don't mind that then should I make a PR for it? 😅

**Comment by Dark-System** [comment]
@hryxna Yeah.. you don't need my approval to make PR. I'll review them and label them Hacktoberfest-accepted if they are aligning with the rules. But make sure you don't spam.
